File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_02_Exercise/_01_Ages/_01_Ages.cs
using System;

public class _01_Ages
{
    public static void Main(string[] args)
    {
        // Raed input value
        int n = int.Parse(Console.ReadLine());

        // Print result based on the input
        if (n <= 2)
        {
            Console.WriteLine("baby");
        }
        else if (n <= 13)
        {
            Console.WriteLine("child");
        }
        else if (n <= 19)
        {
            Console.WriteLine("teenager");
        }
        else if (n <= 65)
        {
            Console.WriteLine("adult");
        }
        else 
        {
            Console.WriteLine("elder");
        }
    }
}

File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_02_Exercise/_01_Ages.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._02_Exercise;

import java.util.Scanner;

public class _01_Ages {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Print result based on the input
        if (n <= 2) {
            System.out.println("baby");
        } else if (n <= 13) {
            System.out.println("child");
        } else if (n <= 19) {
            System.out.println("teenager");
        } else if (n <= 65) {
            System.out.println("adult");
        } else {
            System.out.println("elder");
        }

        scan.close();
    }
}
